c++/cpp2py/converters/vector.hpp

```diff
@@ -5,63 +5,60 @@
 
 #include "../macros.hpp"
 #include "../numpy_proxy.hpp"
+#include "../py_converter.hpp"
 
 namespace cpp2py {
 
   template <typename T> static void delete_pycapsule(PyObject *capsule) {
-    auto *ptr = static_cast<std::unique_ptr<T[]> *>(PyCapsule_GetPointer(capsule, "guard"));
+    auto *ptr = static_cast<T *>(PyCapsule_GetPointer(capsule, "guard"));
```

Comment:
Move the deleter in general cpp2py file

Reply:
How about we just doing
```c++
auto delete_pycapsule = [](PyObject *capsule) {
          auto *ptr = static_cast<std::vector<T> *>(PyCapsule_GetPointer(capsule, "guard"));
          delete ptr;
        };
PyObject * capsule = PyCapsule_New(vec_heap, "guard", delete_pycapsule);
```
right where we need it?